content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -56,13 +56,13 @@ A cluster administrator creates a number of PVs. They carry the details of the r
 #### Dynamic
 When none of the static PVs the administrator created matches a user's `PersistentVolumeClaim`,
 the cluster may try to dynamically provision a volume specially for the PVC.
-This provisioning is based on `StorageClasses`: the PVC must request a
+This provisioning is based on `StorageClass` objects: the PVC must request a
 [storage class](/docs/concepts/storage/storage-classes/) and
 the administrator must have created and configured that class in order for dynamic
 provisioning to occur. Claims that request the class `""` effectively disable
 dynamic provisioning for themselves.
 
-To enable dynamic storage provisioning based on storage class, the cluster administrator
+To enable dynamic storage provisioning based on `StorageClass`, the cluster administrator
```

Comment:
This reads like a prerequisite and should be introduced earlier. "You must enable.... before enabling...."